readme.md

```diff
@@ -220,10 +220,11 @@ Integration tests are run under the `it` profile with the Failsafe plugin using
 
 ## Changelog
 
-### 3.2.0 (Unreleased)
+### 3.2.0 (24 March 2022)
 - The minimum version of Jenkins changed to be **2.289.1**
 - Fix JENKINS-66539 we now handle rate limiting for posting build statuses
 - JENKINS-66498 BitbucketSCMStep can now use serverName instead of serverId to specify a Bitbucket instance
+- Small bug and security fixes
```

Comment:
Does this need a Jenkins security advisory?

Reply:
Yes, we're in touch with the Jenkins security team and will cite that advisory in our release notes once it's been released.

Reply:
Advisory provided here: https://github.com/jenkinsci/atlassian-bitbucket-server-integration-plugin/pull/341